### PR TITLE
SHOOTER: Control from ros services

### PR DESCRIPTION
**Pull request by ironmig**
As per #24, the shooter can be controlled with a service.

* A std_srvs/Trigger call to /shooter/fire will fire balls (currently all 4, will change with new loader)

* A std_srvs/Trigger call to /shooter/cancel will stop any movement on the shooter

* A custom service can manually control the two motors on the shooter for testing

* The arduino ros client has also been added to the motor controller launch file so it gets started when the boat starts up
* Launch file for arduino client has been updated to use the correct USB port ID

### CODE REVIEW

**Comment by ironmig** [comment]
I'll look into the CI error. Perhaps it didn't rebuild the CMAKE stuff to see the new service file? By the way this code was tested on the lake on 8/20 and worked as expected.

**Comment by DSsoto** [comment]
Not sure what it is, but it's not that. Semaphore downloads the repo and builds from scratch every time.

**Comment by mattlangford** [comment]
Not sure if you have access to see where the build failed but this is the message:
```
/home/runner/navigator_ws/src/Navigator/mission_systems/shooter/firmware/shooter.cpp:5:42: fatal error: navigator_msgs/ShooterManual.h: No such file or directory
 #include <navigator_msgs/ShooterManual.h>
```

**Comment by mattlangford** [comment]
After lake day I should have some free time that we can sit down and diagnose the issues here
I don't want to merge yet, since any subsequent builds would fail too

**Comment by mattlangford** [comment]
Semaphore looks like it builds correctly now (though it won't with an old `install.sh`). If you guys want to bring this up to the latests master, we will merge

File: mission_control/navigator_launch/launch/motor_controller.launch [hunk 1]
```diff
@@ -15,6 +15,9 @@
     <param name="port" value="/dev/serial/by-id/usb-FTDI_Roboteq_MDC1460_stbd-bow-if00-port0"/>
     <param name="value" value="115200" />
   </node>
+  <node pkg="rosserial_python" type="serial_node.py" name="arduino_serial">
```
Comment (mattlangford) [inline mission_control/navigator_launch/launch/motor_controller.launch]: Can you make a new `hw_drivers.launch` and put this in there? We will put the hydrophones launch in there and any other hardware interfaces (excluding the motor drivers).

File: mission_control/navigator_launch/launch/master_boat.launch [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <launch>
     <include file="$(find navigator_launch)/launch/motor_controller.launch"/>
+    <include file="$(find navigator_launch)/launch/hw_drivers.launch"/>
```
Comment (mattlangford) [inline mission_control/navigator_launch/launch/master_boat.launch]: Good catch

**Pull request merged**